app/src/tracing.h

```diff
@@ -83,17 +83,19 @@ namespace scitt
 
   static void log_request_end(
     const std::shared_ptr<ccf::RpcContext>& rpc_ctx,
-    const std::string& method,
+    const std::string& path,
     const std::function<ccf::ApiResult(::timespec& time)>& get_time,
     std::optional<ccf::TxID> txid = std::nullopt)
   {
     AppData& app_data = get_app_data(rpc_ctx);
 
     ::timespec end;
     ccf::ApiResult result = get_time(end);
+    // FIXME: should be CBOR or JSON
```

Comment:
Does that need to be done before merging, or is that separate?

Reply:
Technically speaking it should be done now

Reply:
Replaced with explicit CBOR error